packages/web/src/Components/GridContainer.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FunctionComponent } from 'react';
+
```

Comment:
Remove this line

Reply:
Remove? But GridContainder depends on them